### ISSUE TITLE
Bug: TypeError: Cannot read property 'get' of undefined

### ISSUE THREAD

**Issue by blakewilson** [wpengine/faustjs#640]
When spinning up a new `examples/next/getting-started` example, you may get the following error: 

```
Server Error

TypeError: Cannot read property 'get' of undefined
This error happened while generating the page. Any console logs will be displayed in the terminal window.
Call Stack
_interopNamespace
file:/Users/blakewilson/Desktop/faust-walkthrough/node_modules/@gqty/react/ssr/ssr.js (15:39)
<unknown>
file:/Users/blakewilson/Desktop/faust-walkthrough/node_modules/@gqty/react/ssr/ssr.js (30:88)
async Object.prepareReactRender2 [as prepareReactRender]
file:/Users/blakewilson/Desktop/faust-walkthrough/node_modules/@gqty/react/ssr/ssr.js (30:25)
async eval
node_modules/@faustjs/next/dist/mjs/server/getProps.js (19:50)
async Object.prepareRender
file:/Users/blakewilson/Desktop/faust-walkthrough/node_modules/gqty/Helpers/ssr.js (39:7)
async getProps
node_modules/@faustjs/next/dist/mjs/server/getProps.js (18:50)
async getNextStaticProps
node_modules/@faustjs/next/dist/mjs/server/getProps.js (155:0)
```

## Applicable Versions

- `@faustjs/core` version: 0.12.4
- `@faustjs/react` version: N/A
- `@faustjs/next` version: 0.13.0
- WordPress version: 5.8.1

**Comment by blistok** [comment]
FWIW, I only experience this error if permalinks are not set to:
```
/posts/%postname%/
```
as described in [the instructions](https://faustjs.org/docs/next/getting-started#setting-your-permalinks).

**Comment by blakewilson** [comment]
This issue is happening due to the use of `sass` in `examples/next/getting-started`. `sass` is built using [Dart](https://dart.dev/), [which mutates `Function.prototype`](https://github.com/sass/dart-sass/issues/1224).

This is ultimately causing a race condition between [`prepareReactRender`](https://gqty.dev/docs/react/ssr#preparereactrender) in [GQty](https://gqty.dev/), and the compilation of the SCSS files.

Below is the module resolution of `react-ssr-prepass` in [`@gqty/react/ssr/ssr.js`](https://github.com/gqty-dev/gqty/blob/main/packages/react/src/ssr/ssr.ts#L50) upon a sucessful page load:

![Screen Shot 2021-11-11 at 3 30 38 PM](https://user-images.githubusercontent.com/5946219/141373168-65b3948f-cf26-4a17-8a50-1265bc1b8719.png)

Next, here is the module resolution when the race condition is not met, and `sass` alters the `Function.prototype` of `react-ssr-prepass`:
![Screen Shot 2021-11-11 at 3 29 27 PM](https://user-images.githubusercontent.com/5946219/141373353-16f0d221-ab7c-4cf7-9382-07069903d9a5.png)

Notice the mangled prototype as described in this issue: https://github.com/sass/dart-sass/issues/1224

Finally, the runtime error `TypeError: Cannot read property 'get' of undefined` is encountered in the `_interopNamespace` function generated by TypeScript in `@gqty/react/ssr/ssr.js`:

![Screen Shot 2021-11-11 at 3 46 06 PM](https://user-images.githubusercontent.com/5946219/141373631-2bf5bb05-c4fe-491e-a904-73d2d6151281.png)

In my screenshot above, `d` is `undefined`, thus, the property `get` is not accesible in the next line.

**Comment by blakewilson** [comment]
In the meantime, a temporary alternative is to swap your [`sass`](https://www.npmjs.com/package/sass) dependency with [`node-sass`](https://www.npmjs.com/package/node-sass) in `examples/next/getting-started`, as `node-sass` was not written in Dart.

However, this should only be a temporary workaround until a fix is in place as `node-sass` has been deprecated in favor of `sass`.

**Comment by blakewilson** [comment]
@PabloSzx Have you experienced this before with `prepareReactRender`?

**Comment by PabloSzx** [comment]
I will take a look, should be easy to fix

**Comment by PabloSzx** [comment]
Should be fixed in @gqty/react@2.1.0 🎉

**Comment by blakewilson** [comment]
Amazing! Thanks for the quick fix @PabloSzx!

**Comment by blakewilson** [comment]
Fixed in `0.13.1` via https://github.com/wpengine/faustjs/pull/642
